src/font.js

```diff
@@ -295,7 +295,7 @@ Font.prototype.forEachGlyph = function(text, x, y, fontSize, options, callback)
     x = x !== undefined ? x : 0;
     y = y !== undefined ? y : 0;
     fontSize = fontSize !== undefined ? fontSize : 72;
-    options = options || this.defaultRenderOptions;
+    options = options ? Object.assign(options, this.defaultRenderOptions) : this.defaultRenderOptions;
```

Comment:
This line looks backwards to me; it will overwrite any given options with the default options, rather than the reverse (it will also mutate the passed in options object, though that usually won't matter). Really it should be:

```
options = options ? Object.assign({}, this.defaultRenderOptions, options) : this.defaultRenderOptions;
```

Which can be even simpler, since Object.assign can handle nulls/undefineds:

```suggestion
    options = Object.assign({}, this.defaultRenderOptions, options);
```